templates/scripts.html

```diff
@@ -1,4 +1,5 @@
 <script src="{{ url_for('prompts.static', filename='js/utils/tableEvents.js') }}"></script>
+<script src="https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.min.js"></script>
```

Comment:
we need to keep external libraries locally

Reply:
Of course, where can I move this?